apps/ledger-live-desktop/src/renderer/families/tezos/DelegateFlowModal/Body.tsx

```diff
@@ -107,15 +93,15 @@ const createSteps = (params: Data): St[] => [
     footer: StepConfirmationFooter,
   },
 ];
-const Body = ({ onChangeStepId, onClose, stepId, params }: Props) => {
+
+const Body = ({ stepId, params, onChangeStepId, onClose }: Props) => {
   const { t } = useTranslation();
   const dispatch = useDispatch();
   const device = useSelector(getCurrentDevice);
-  const accounts = useSelector(delegatableAccountsSelector);
   const openedFromAccount = !!params.account;
-  const bakers = useBakers(whitelist);
-  const randomBaker = useRandomBaker(bakers);
-  const [steps] = useState(() => createSteps(params));
```

Comment:
I'm not very confident in react, why did we remove this useState ?

Reply:
useState is used to manage a state with react

`const [state, setState] = useState(() => defaultValue)`
If you don't need to update the state with a `setState` then the value never changes so it's juste a constant